### ISSUE TITLE
TraCeR on 10X cellranegr vdj output filtered_contig.fasta demultiplexed files

### ISSUE THREAD

**Issue by arutik** [Teichlab/tracer#95]
Hi,

Similar to this issue https://github.com/Teichlab/bracer/issues/21#issuecomment-417711510 in BraCeR, can one run TraCeR on the per-cell fasta files demultiplexed from filtered_contig.fasta output of cellranger vdj?
If yes, then with what adjustments?

Thank you.

**Comment by mstubb** [comment]
Yes, I think that should enable you to get the files through TraCeR (although I've not tried it). Is there a reason you want to do that rather than using the Cell Ranger annotations?

**Comment by arutik** [comment]
Thank you!

I first wanted to do this for BraCeR because I find BraCeR output data more comprehensive (for example, cellranger vdj doesn't output IG isotype distribution or I can't find it) so I assumed the same would apply to TraCeR vs cellranger vdj, but I guess here there should be no major difference... or is there any TraCeR-specific output?

**Comment by mstubb** [comment]
I'm not sure it's going to be worth it. The main addition that tracer has is the clonotype graphs but those become too resource-intensive to draw once you get beyond a few hundred cells so I don't think you'll gain much. 

All the best,

Mike

**Comment by arutik** [comment]
Hi, 

Coming back to the previous discussion, do you know if cellranger vdj reports CDR3 region sequences like TraCeR and BraCeR do?

Thank you.

Sincerely,
Anna Arutyunyan

**Comment by mstubb** [comment]
Hi Anna,

Yes, the CDR3 sequences are in the Cell Ranger annotation file outputs (https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/output/annotation). 

Note that these are the genuine CDR3 sequences. Tracer reports the ‘junction’ sequences as reported by IgBlast. These are _not_ the canonical CDR3s. So, Cell Ranger’s output is probably more useful. 

Cheers,

Mike

**Comment by arutik** [comment]
Thank you very much Mike!

Sincerely, 
Anna Arutyunyan

**Comment by korakusa** [comment]
Hi,
I also would like to run TraCer with 10x genomics data (filtered_contig.fasta output of cellranger vdj) because I'm so interested in TraCer clonotype network graph.
I referred to  #Teichlab/bracer21 and split my fasta file.
But in TraCer, there is no --assembled_file option.
How could I run tracer assemble?

Sincerely,
Sakurako
